README.md

```diff
@@ -13,26 +13,46 @@ itself.
 ![Dashboard UI workloads page](docs/dashboard-ui.png)
 
 ## Deployment
-It is likely that the Dashboard is already installed on your cluster. Check with the following command:
+Provided instructions are compliant with Kubernetes 1.6+. If you plan to deploy Dashboard on older Kubernetes version please
+follow [these instructions](docs/user-guide/deployment-old.md) or if you want to check out latest
+features that the team works on try our HEAD releases [installation guide](docs/devel/head-releases.md).
+
+For security reasons installation has been split into two parts:
+[Deployment](#deployment) and [Configuration](#configuration). Deployment part will deploy Dashboard on your
+cluster without any privileges. It will only work if RBACs are disabled on your cluster.
```

Comment:
This is the current behavior but my proposal is to split insallation and privileges configuration into 2 parts so simple `kubectl create -f dashboard.yaml` would deploy dashboard but without cluster role or service account with admin privileges.

Then as a second step we'd deploy access rights configuration based on chosen configuration: minimal or full.